### ISSUE TITLE
conv5_3 layers contains NANs causing SIGFPE

### ISSUE THREAD

**Issue by leduckhc** [daijifeng001/MNC#53]
Hi. I was examining the cause of SIGFPE 8 when fine-tuning the network for VOCpascalSDS from pre-trained.VGG16 model. For me, yet not obvious reasons, in 3rd iteration the weights of `conv5_3` (and almost all the layers below `conv5_{1,2}, conv4_{1,2,3}, conv3_{1,2,3}`, etc) contains NANs. This cause  unpredictable results for RPN, Roiwarping, ProposalLayer. It may also be the reason for SIGFPE 8, `np.exp` overflows etc.

I examined by looking at 
```
print {k: v.data for k, v in self.solver.net.blobs.items()}
print {k: v[0].data for k, v in self.solver.net.params.items()}
# v[0] are weights, v[1] are biases
```

**Comment by leduckhc** [comment]
So it turns out, the solution was quite easy. Since it is fine-tuning process the learning rate of ConvNet (VGG16, ZFNet, LeNet) must be set to zero. Just put down `lr_mult=0` for all parameters of the ConvNet layers, for example `conv5_{1,2,3}, conv4_{1,2,3}, conv3_{1,2,3}` of VGG16
```
  param { # weights
    lr_mult: 0
    decay_mult: 0
  }
  param { # biases
    lr_mult: 0
    decay_mult: 0
  }
```
